### PR TITLE
Fix/add cts env

### PR DESCRIPTION
**Pull request by nettashviro**

### CODE REVIEW

File: src/components/layout/sidenav/Sidenav.vue [hunk 1]
```diff
@@ -33,7 +33,7 @@
         <div v-if="externalNetworkDests">
           <router-link
             :to="`/external-transferred-${externalNetworkDest.appID}`"
-            v-for="externalNetworkDest in externalNetworkDests"
+            v-for="externalNetworkDest in externalNetworkDests.filter((networkDest) => networkDest.isEnabled)"
```
Comment (yoavrozner) [inline src/components/layout/sidenav/Sidenav.vue:36]: `enableNetworks: function() {
      return this.externalNetworkDests.filter((networkDest) => networkDest.isEnabled);
    },`

File: src/components/popups/external-transfer-popup/steps/ExternalNetwork.vue [hunk 2]
```diff
@@ -2,9 +2,9 @@
   <div>
     <p class="popup-text">{{ $t("externalTransfer.NetworkDestChoose") }}</p>
 
-    <div class="select-container select-btn mx-auto">
+    <div v-bind:class="[enableNetworks.length == 1 ? 'mx-auto select-btn-one' : 'select-btn', 'select-container']">
```
Comment (yoavrozner) [inline src/components/popups/external-transfer-popup/steps/ExternalNetwork.vue:5]: Very nice!

File: src/router/index.js [hunk 3]
```diff
@@ -133,8 +134,16 @@ const router = new Router({
 });
 
 router.beforeEach(async (to, from, next) => {
+  if (
+    to.path.startsWith("/external-transferred") &&
+    !getNetworkItemByAppId(to.path.split("-").slice(-1)[0]).isEnabled
```
Comment (yoavrozner) [inline src/router/index.js:139]: Like

Comment (yoavrozner) [reply src/router/index.js:139]: But please add docs

File: src/store/modules/configuration.js [hunk 4]
```diff
@@ -28,14 +28,7 @@ const state = {
...
       isDefault: true,
-    },
-    {
-      appID: "appId2",
-      approvalUIUrl: "http://approval.dropbox.rabaz.org",
-      approvalUrl: "http://approval.dropbox.rabaz.org",
-      label: "label2",
-      value: "value2",
-      isDefault: false,
+      isEnabled: true,
```
Comment (yoavrozner) [inline src/store/modules/configuration.js:31]: For next time, use `!-!+!undefined`

**Review by yoavrozner — APPROVED** [review]
Well done!

File: src/utils/networkDest.js [hunk 5]
```diff
@@ -13,3 +13,18 @@ export function getNetworkItemByAppId(appId) {
   )[0];
   return networkDest;
 }
+
+export function getEnabledNetworks() {
+  // Check if network enabled
+  let networkDests = store.state.configuration.externalNetworkDests.filter((networkDest) => networkDest.isEnabled);
```
Comment (yoavrozner) [inline src/utils/networkDest.js:19]:
```suggestion
  let networkDests;
  try {
    networkDests = store.state.configuration.externalNetworkDests.filter((networkDest) => networkDest.isEnabled);
  } catch (_err) {
    networkDests = [];
  }
```

**Review by yoavrozner — APPROVED** [review]
Well done!

File: src/utils/networkDest.js (hunk 5 above)
Comment (nettashviro) [reply src/utils/networkDest.js]: done

**Review by yoavrozner — APPROVED** [review]

**Pull request merged**